lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -1144,6 +1144,11 @@ namespace Js
 
     uint SourceTextModuleRecord::GetLocalExportSlotIndexByExportName(PropertyId exportNameId)
     {
+        if(!this->WasDeclarationInitialized())//allow for circular imports that reference each other's functions
```

Comment:
Nit: coding style isnt a big deal for us, but I think we usually avoid using `this->` unless it is to avoid ambiguity.

Reply:
Happy to remove though this-> is used 109 times in SourceTextModuleRecord so I was kinda copying what I'd seen there.